charts/ndb-operator/templates/service-ndb-operator-controller-manager-metrics-service.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     protocol: TCP
     targetPort: https
   selector:
-    control-plane: {{ include "ndb-operator.fullname" . }}-controller-manager
+    control-plane: controller-manager
```

Comment:
Was this an intended change?

Reply:
I guess not, since "ndb-operator" i.e. namespace name is not included in the text, we can skip adding it there. Removing it now. I will test and confirm this as a part of testing.